bluemarket/src/App.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import Topbar from './components/Topbar.jsx';
 
 function App() {
   return (
     <div className="App">
+            <Topbar/>
```

Comment:
Is there are reason as to why this is indented like this? It is in this div but nothing else? Just need clarification if this is the only tag in this div

Reply:
it was an indentation error it should be in line wit the header below.
fixed